### PR TITLE
Adds custom item (e UMi Labcoat - Looping Song - Devildabeast)

### PR DESCRIPTION
**Pull request by ghost**
Acquired approval from ThatOneGuy in private.

### CODE REVIEW

File: config/custom_items.txt [hunk 1]
```diff
@@ -508,4 +508,14 @@ item_path: /obj/item/clothing/suit/storage/toggle/bomber
...
\ No newline at end of file
+}
+
+{
+ckey: devildabeast
+character_name: Looping Song
+item_path: /obj/item/clothing/suit/storage/labcoat
+item_name: e UMi labcoat
+item_icon: labcoat_black_open
+item_desc: A suit that protects against minor chemical spills. Has a black stripe on the shoulder. The abbreviation \"e UMi\" is written on the back in bold text.
```
Comment (mwerezak) [inline config/custom_items.txt]: BYOND won't respect your escape sequence when ingesting raw text like this. Just use plain quotation marks, like the other items. It splits at the : character and uses the character data on the RHS as is so you don't need to escape anything.

**Comment by ghost** [comment]
I really have no idea what you're talking about. I might need someone else to do this for me.

**Comment by mwerezak** [comment]
...You remove the \ that you put in front of the ".

I thought if you put them there in the first place you would know what I was talking about.

But, okay.

**Comment by ghost** [comment]
I didn't put them there. That's how it came. I'm just trying to delete this now but I can't.

**Comment by mwerezak** [comment]
How it came?

**Comment by mwerezak** [comment]
Well, seeing as you've fixed it I guess you can reopen this whenever you feel like having your custom item again.

**Pull request closed without merging**